CODEOWNERS

```diff
@@ -65,7 +65,7 @@
 /specification/service-map/ @daveirwin1
 /specification/servicebus/ @sazeesha @v-ajnava
 /specification/servicefabric/ @juhacket @samedder
-/specification/sql/ @dnayantara @jaredmoo @nathannfan
+/specification/sql/ @dnayantara @jaredmoo @nathannfan @jamestao
```

Comment:
Nayantara and Nathan have both left MSFT, so let's remove them ;)

Reply:
If you don't remove Jared name then ARM team might assign PR to Jared.